src/config.js

```diff
@@ -13,6 +13,7 @@ const config = {
     useSSL: (process.env.EPSAGON_SSL || 'TRUE').toUpperCase() === 'TRUE',
     traceCollectorURL: consts.TRACE_COLLECTOR_URL,
     isEpsagonDisabled: (process.env.DISABLE_EPSAGON || '').toUpperCase() === 'TRUE',
+    urlPatternsToIgnore: [],
```

Comment:
Let add here the ability to send params through env as well (I added this to Python as well).
the name should be `EPSAGON_URLS_TO_IGNORE` and it should include comma-separated patterns like
`book.com,la.co.il`